### ISSUE TITLE
Allow ActionItems to be directly specified between the ActionBar opening and closing tags.

### ISSUE THREAD

**Issue by PanayotCankov** [NativeScript/NativeScript#518]
The silly me put this on a page:
``` XML
  <Page.actionBar>
    <ActionBar>
      <ActionItem text="{{ resources.signUp }}" tap="signUpMenuItemTap" android.position="actionBar" />
    </ActionBar>
  </Page.actionBar>
```

At runtime it works like a charm. The only problem is the action item does not appear.

Obviously the correct usage is:
``` XML
  <Page.actionBar>
    <ActionBar>
      <ActionBar.actionItems>
        <ActionItem text="{{ resources.signUp }}" tap="signUpMenuItemTap" android.position="actionBar" />
      </ActionBar.actionItems>
    </ActionBar>
  </Page.actionBar>
```
The problem is the first case does not generate neither compile-time nor run-time error.

**Comment by ligaz** [comment]
See [here](https://github.com/NativeScript/NativeScript/issues/534) for a possible solution

**Comment by PanayotCankov** [comment]
I would agree to have both decorators marking the content properties and errors/warnings. However when you look at the issues - the snippets are identical but the problems are different.

**Comment by enchev** [comment]
Merged with this: #597
